packages/core/src/electron-browser/menu/electron-main-menu-factory.ts

```diff
@@ -74,57 +74,58 @@ export type ElectronMenuItemRole = ('undo' | 'redo' | 'cut' | 'copy' | 'paste' |
 @injectable()
 export class ElectronMainMenuFactory extends BrowserMainMenuFactory {
 
-    protected _menu?: MenuDto[];
-    protected _toggledCommands: Set<string> = new Set();
+    protected menu?: MenuDto[];
+    protected toggledCommands: Set<string> = new Set();
 
     @inject(PreferenceService)
     protected preferencesService: PreferenceService;
 
+    setMenuBar = debounce(() => this.doSetMenuBar());
```

Comment:
Minor: Having an explicit delay value here would be good. I never know, what the default for this is.
```suggestion
    setMenuBar = debounce(() => this.doSetMenuBar(), 10);
```

Reply:
@msujew I upped the value to 100, since updating the main menu 100 times per second seems a bit excessive ;-)